jenkinsfiles/Jenkinsfile.nightly

```diff
@@ -133,6 +133,7 @@ pipeline {
     }
     post {
         always {
+            sh 'lscpu'
```

Comment:
I note that some lines use double-quote and others use single-quote, but seems like both are respected.